index.html

```diff
@@ -1412,6 +1420,16 @@ <h2>Public Keys</h2>
   "id": "did:example:123456789abcdefghi",
   <span class="comment">...</span>
   "publicKey": [{
+    "id": "did:example:123#_Qq0UL2Fq651Q0Fjd6TvnYE-faHiOpRlPVQcY_-tA4A",
```

Comment:
I really like this as a best practice for JWK `kid`s in DID documents.

Reply:
can I resolve this?